tensorboard/webapp/runs/store/runs_reducers.ts

```diff
@@ -95,7 +96,7 @@ const dataReducer: ActionReducer<RunsDataState, Action> = createReducer(
 
     return {
       ...state,
-      userSetGroupBy: groupBy,
+      userSetGroupByKey: groupBy.key ?? undefined,
```

Comment:
> `userSetGroupByKey: groupBy.key ?? undefined,`

Can `groupBy.key` be `undefined`? If so, change this to `userSetGroupByKey: groupBy.key ?? null` instead.